### PR TITLE
Test what happens when removing remote github R package downloads

### PR DESCRIPTION
**Pull request by tonygardella**
OK to MERGE
## Description
- Removed `geonames` and `nneo` from remotes in data.atmosphere DESRIPTION as they are on CRAN.
- Try installing linkages and preles models  by git cloning them travis.yml and building them. Removed the Remote call in their respective DESCRIPTION file.
- Rename the DESCRIPTION file in the elicitation and pecan.depend shiny apps so the install shiny script can't find their dependency packages.

## Motivation and Context
What to see if the rate limit is still hit if reduce some of the Github api calls.
## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [ ] When possible
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/data.atmosphere/DESCRIPTION [hunk 1]
```diff
@@ -57,12 +57,11 @@ Imports:
 Suggests:
     doParallel,
     foreach,
+    geonames,
+    nneo,
     parallel,
     progress,
     rlang (>= 0.2.0)
-Remotes:
-    github::ropensci/geonames,
```
Comment (infotroph) [inline modules/data.atmosphere/DESCRIPTION]: We still need the GitHub version geonames until they do another CRAN release -- see #1941 for details.

File: modules/data.atmosphere/DESCRIPTION [hunk 2]
```diff
@@ -57,12 +57,11 @@ Imports:
 Suggests:
     doParallel,
     foreach,
+    geonames,
+    nneo,
```
Comment (infotroph) [inline modules/data.atmosphere/DESCRIPTION]: No need to add these here -- They're already in Imports above.

File: modules/data.atmosphere/DESCRIPTION (hunk 1 above)
Comment (infotroph) [reply modules/data.atmosphere/DESCRIPTION]: Come to think of it this means we should be specifying the import as `geonames > 0.998`, but they haven't yet bumped the version string in the dev version. I've submitted a PR to geonames and will update here if/when they accept it.

File: modules/data.atmosphere/DESCRIPTION [hunk 3]
```diff
@@ -57,12 +57,11 @@ Imports:
...
     parallel,
     progress,
     rlang (>= 0.2.0)
-Remotes:
-    github::ropensci/geonames,
-    github::ropenscilabs/nneo
```
Comment (infotroph) [inline modules/data.atmosphere/DESCRIPTION]: Right now the CRAN version of nneo is 0.1.0 and the dev version is 0.1.1.9313, but I don't know whether PEcAn needs any of the changes.

If keeping github version, note that repo has moved to `ropensci/nneo` (no "labs")

File: models/linkages/DESCRIPTION [hunk 4]
```diff
@@ -19,6 +19,8 @@ Imports:
 Suggests:
     testthat (>= 1.0.2),
     linkages
+Remotes:
+  github::ariaho/linkages_package
```
Comment (infotroph) [inline models/linkages/DESCRIPTION]: typo

**Comment by tonygardella** [comment]
@infotroph Let me know what you're thinking for geonames and I can implement. Seems like we can go back to having PRs form personal forks now.

File: .travis.yml [hunk 5]
```diff
@@ -105,7 +105,21 @@ install:
...
+  - git clone https://github.com/MikkoPeltoniemi/Rpreles
+  - cd Rpreles
+  - R CMD INSTALL .
+  - popd
+  - pushd $HOME
+  # Install linakages package
+  - git clone https://github.com/araiho/linkages_package
+  - cd linkages_package
+  - R CMD INSTALL .
+  - popd
```
Comment (infotroph) [inline .travis.yml]: Let's go back to installing these in DESCRIPTION

File: modules/data.atmosphere/DESCRIPTION [hunk 6]
```diff
@@ -23,7 +23,6 @@ Imports:
     car,
     data.table,
     dplyr,
-    geonames,
```
Comment (infotroph) [inline modules/data.atmosphere/DESCRIPTION]: Still need this line whether installing from CRAN or GitHub

File: modules/data.atmosphere/DESCRIPTION [hunk 7]
```diff
@@ -34,7 +33,6 @@ Imports:
     MASS,
     mgcv,
     ncdf4 (>= 1.15),
-    nneo,
```
Comment (infotroph) [inline modules/data.atmosphere/DESCRIPTION]: Still need this line whether installing from CRAN or GitHub

File: modules/data.atmosphere/DESCRIPTION [hunk 8]
```diff
@@ -61,8 +59,8 @@ Suggests:
     progress,
     rlang (>= 0.2.0)
 Remotes:
-    github::ropensci/geonames,
-    github::ropenscilabs/nneo
+    github::ropensci/geonames ,
```
Comment (infotroph) [inline modules/data.atmosphere/DESCRIPTION]: Extra space before comma

File: .travis.yml [hunk 9]
```diff
@@ -105,7 +105,21 @@ install:
...
+  #install remotes if version less than or equal 2.0.1
```
Comment (infotroph) [inline .travis.yml]: More than the version number (most folks can probably infer that from the code below), would probably be good to record the reason. Maybe `# temporary until remotes 2.0.2 binary reaches C2D4U`?

File: modules/data.atmosphere/DESCRIPTION (hunk 1 above)
Comment (infotroph) [reply modules/data.atmosphere/DESCRIPTION]: geonames version bump PR: https://github.com/ropensci/geonames/pull/18

File: modules/data.atmosphere/DESCRIPTION [hunk 10]
```diff
@@ -61,8 +59,8 @@ Suggests:
...
+    github::ropensci/nneo
```
Comment (infotroph) [inline modules/data.atmosphere/DESCRIPTION]: While we're changing this, should update https://github.com/PecanProject/pecan/blob/develop/docker/base/Dockerfile.depends#L22 to match (cc @robkooper)

**Comment by tonygardella** [comment]
Thank you @infotroph!

**Pull request merged**